db/migrations/postgres/000012_create_users.up.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE IF NOT EXISTS users (
+    id VARCHAR(26) PRIMARY KEY,
+    createat bigint,
+    updateat bigint,
+    deleteat bigint,
+    username VARCHAR(64) UNIQUE,
+    password VARCHAR(128),
+    authdata VARCHAR(128) UNIQUE,
+    authservice VARCHAR(32),
+    email VARCHAR(128) UNIQUE,
+    emailverified boolean,
+    nickname VARCHAR(64),
+    firstname VARCHAR(64),
+    lastname VARCHAR(64),
+    roles VARCHAR(256),
+    allowmarketing boolean,
+    props VARCHAR(4000),
+    notifyprops VARCHAR(2000),
+    lastpasswordupdate bigint,
+    lastpictureupdate bigint,
+    failedattempts integer,
+    locale VARCHAR(5),
+    mfaactive boolean,
+    mfasecret VARCHAR(128)
+);
+
+ALTER TABLE users DROP COLUMN IF EXISTS lastactivityat;
+ALTER TABLE users DROP COLUMN IF EXISTS lastpingat;
+ALTER TABLE users ADD COLUMN IF NOT EXISTS position VARCHAR(128);
+ALTER TABLE users ADD COLUMN IF NOT EXISTS timezone VARCHAR(256) DEFAULT '{"automaticTimezone":"","manualTimezone":"","useAutomaticTimezone":"true"}';
+ALTER TABLE users ALTER COLUMN position TYPE VARCHAR(128);
```

Comment:
is this a noop if the column has already this type? shouldn't we do something like this? https://github.com/mattermost/mattermost-server/pull/17463/files#diff-e01705889e6c5b9d74cfdf3879cd43b22aa506a5a89d11ee9fd0a8d3520cd07dR13